docs-requirements.txt

```diff
@@ -1,11 +1,10 @@
-jinja2 == 3.0.3
```

Comment:
might need to pin this still until jinja resolves, still getting: `ImportError: cannot import name 'environmentfilter' from 'jinja2' (/home/docs/checkouts/readthedocs.org/user_builds/feature-labs-inc-evalml/envs/3411/lib/python3.8/site-packages/jinja2/__init__.py)`

Reply:
looks like an old version of sphinx